CMakeLists.txt

```diff
@@ -16,8 +16,6 @@ project("Woof"
     VERSION 12.0.0
     LANGUAGES C)
 
-set(CMAKE_C_STANDARD 99)
```

Comment:
Could you move back to where it was? It was actually doing its job, we just set this extra warning.

Reply:
Does it work? I think we should use https://cmake.org/cmake/help/latest/manual/cmake-compile-features.7.html#manual:cmake-compile-features(7)

Reply:
Yes, it does. It sets the default value for the `C_STANDARD` feature, which provides a fallback mechanism for compilers which doe not support specific feature sets:

https://cmake.org/cmake/help/latest/prop_tgt/C_STANDARD.html

In our case, it indeed adds e.g. the `-std=gnu99` flag to the GCC call.

Reply:
We should probably even `set(CMAKE_C_STANDARD_REQUIRED ON)`. 🤔